src/app/cluster/details/cluster/gatekeeper-config/gatekeeper-config-dialog/component.spec.ts

```diff
@@ -99,7 +99,6 @@ describe('GatekeeperConfigDialog', () => {
     });
 
     it('should call createGatekeeperConfig()', () => {
-      component.save();
```

Comment:
This unit test does not make much sense after removing this (same for the patch one). Without any action, it assumes that create will be called. It should be a result of some kind of action like a simulated click on the button or direct call to a method.